src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -118,6 +118,18 @@
       + "provide fixed width filenames that can be ordered by simple lexicographic sorting.";
   public static final int FILENAME_OFFSET_ZERO_PAD_WIDTH_DEFAULT = 10;
 
+  public static final String SECURE_HADOOP_CONFIG = "is.secure.hadoop";
+  private static final String SECURE_HADOOP_DOC = "Config indicating whether the hadoop cluster is secure or not";
```

Comment:
Maybe we should be more specific? "secure" hadoop can mean a bunch of stuff (including HDFS encryption...). 
We only do Kerberos authentication, so I'd like to be clear on this.